package.json

```diff
@@ -6,26 +6,13 @@
     "./website",
     "./packages/*",
     "./packages/plugins/*",
-    "./examples/apollo-server",
-    "./examples/graphql-ws",
-    "./examples/graphql-sse",
-    "./examples/graphql-socket.io",
-    "./examples/simple-http",
-    "./examples/graphql-helix",
-    "./examples/graphql-helix-auth0",
-    "./examples/lambda-aws",
-    "./examples/azure-functions",
-    "./examples/google-cloud-functions",
-    "./examples/cloudflare-workers",
-    "./examples/with-esm",
-    "./examples/typegraphql",
-    "./examples/nexus",
-    "./examples/express-graphql"
+    "./examples/*"
```

Comment:
please revert this change as we have some examples that are not part of the workspace that break due to dependency hoisting.